### ISSUE TITLE
In repo summary, use labels as present in the repositories?

### ISSUE THREAD

**Issue by maelle** [jimhester/tidyversedashboard#7]
Again, not sure if generally useful and something I did for Locke Data package repos where "bug" is not a label.

I used the labels present in the repositories for the repo summary https://github.com/lockedata/tidyversedashboard/blob/master/R/repository.R#L54 

I also wonder whether labels should be merged when they're not homogeneous. E.g. the repo summary says there's 0 bug for `dplyr` because there the bugs are "bug 💣 ", but having two columns "bug" and "bug :bomb: " might not be very handy. Maybe `num_label` should use a regexp.

Sorry this was a messy issue!

**Comment by maelle** [comment]
Using all labels in the summary is probably not a good idea 🤔 but using a regexp maybe a better one 😁

**Comment by jimhester** [comment]
Yeah, maybe this should just be a parameter to the report of what labels to aggregate by. It could just be a named list if there are ones you need to map to a common category, e.g.

```r
issue_categories <- c("bug 💣" = "bug", "bug" = "bug", "feature" = "feature")
```
